worldengine/model/world.py

```diff
@@ -852,7 +851,67 @@ def is_chaparral(self, pos):
     #
 
     def n_actual_plates(self):
-        return self.layers['plates'].data.max() + 1
+        return self.plates.data.max() + 1
+
+    #
+    # Accessors
+    #
+
+    @property
+    def elevation(self):
+        return self.layers['elevation']
```

Comment:
That last problem can be handled easily.
```python
@elevation.setter
def elevation(self, value):
    if type(value) is not LayerWithThresholds:
        value = LayerWithThresholds(value, None)
    self.layers['elevation'].data = value
```

I am not sure about multiple parameters, though. Doesn't `world.elevation = (data, thresholds)` work?